src/Wizard.js

```diff
@@ -34,6 +37,23 @@ const Wizard = ({ children, onComplete, debug }: Props) => {
     ]);
   }
 
+  function updateHistory(currStepName, nextStepName) {
+    if (typeof window === 'undefined') {
+      return;
+    }
+    const currUrl = window.location.href;
```

Comment:
This is just a personal opinion, but I prefer when full length names rather then shortened ones. So "currentUrl" as opposed to "currUrl".

Reply:
Ok!